core/Enclave/self_test/rsa_test.cpp

```diff
@@ -461,7 +461,7 @@ static ehsm_padding_mode_t get_paddingmode(int paddingmode)
     case 3:
         return EH_PAD_RSA_PKCS1_OAEP;
     default:
-        return EH_PADDING_NONE;
+        return EH_PAD_RSA_NO;
```

Comment:
what's the difference about EH_PADDING_NONE and EH_PAD_RSA_NO, seems the EH_PADDING_NONE is better.

Reply:
delete EH_PAD_RSA_NO, only keep EH_PADDING_NONE